internal/authentication/authentication.go

```diff
@@ -8,6 +8,8 @@ import (
 	"github.com/gpuctl/gpuctl/internal/femto"
 )
 
+const TOKEN_KEY = "token"
```

Comment:
NIT: This should be `TokenCookieName`. https://go.dev/wiki/CodeReviewComments#mixed-caps

Reply:
https://github.com/gpuctl/gpuctl/pull/140/commits/c9500a3b01e70a2199d2b798182a0a7c2e4bfdb3